runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsOverlay.java

```diff
@@ -101,6 +104,28 @@ public Dimension render(Graphics2D graphics)
 			.color(color)
 			.build());
 
+		if (config.ccDisplay())
+		{
+			color = Color.RED;
+			ClanMemberManager clanMemberManager = client.getClanMemberManager();
+			FontMetrics metrics = graphics.getFontMetrics();
+			String worldString = "W" + client.getWorld();
+			String clanOwner = "Open CC tab...";
```

Comment:
I don't think this is dependent upon the tab being open?

Reply:
correct, its a relic of the method that was being used before clanMemeberManager existed, ill just make it "Join a CC" or something.